pkg/autoscaler/statforwarder/forwarder.go

```diff
@@ -253,15 +274,80 @@ func (f *Forwarder) createOrUpdateEndpoints(ns, n string) error {
 	return nil
 }
 
-func (f *Forwarder) getHolder(name string) (string, bool) {
-	f.leaseLock.RLock()
-	defer f.leaseLock.RUnlock()
-	result, ok := f.leaseHolders[name]
+func (f *Forwarder) getProcessor(bkt string) (*bucketProcessor, bool) {
```

Comment:
do we need bool here? If the key isn't there it'll be just nil.

Reply:
done